### PR TITLE
[Cortx1.0-dev] EOS-14605-Added error code description in swagger.json for UDx device, firmware update, hotfix update and SSL certificate

### PR DESCRIPTION
**Pull request by namrata-khake-ui**
# Front-end 

## Problem Statement
<pre>
  <code>
Story Ref (if any): EOS-14605 Swagger UI update with error code description
  </code>
</pre>
## Unit testing on RPM done
<pre>
  <code>
 NOT Required
  </code>
</pre>
## Problem Description
<pre>
  <code>
   Swagger UI update with error code description
  </code>
</pre>
## Solution
<pre>
  <code>
Added error code description to following component

- UDX Device
- Firmware Update
- SSL Certificate
- Hotfix update
- 

- Added or removed description which are not going to applicable for any API

  </code>
</pre>

## Unit Test Cases
<pre>
  <code>
- NOT required
  </code>
</pre>

Signed-off-by: Namrata Khake <namrata.khake@seagate.com>

### CODE REVIEW

**Review by giftedUI — APPROVED** [review]

File: web/src/config/swagger.json [hunk 1]
```diff
@@ -3418,37 +3259,22 @@
...
-				},
-				"403": {
-					"description": "Forbidden"
-				},
-				"404": {
-					"description": "No Data Found"
-				},
-				"409": {
-					"description": "Conflict"
+					"description": "Unauthorized: Token has not found or invalid format or session is invalid"
```
Comment (navalkp) [inline web/src/config/swagger.json:3268]: 403 is required for non public api

File: web/src/config/swagger.json [hunk 2]
```diff
@@ -3147,37 +3063,22 @@
...
-					},
-					"403": {
-						"description": "Forbidden"
-					},
-					"404": {
-						"description": "No Data Found"
-					},
-					"409": {
-						"description": "Conflict"
+						"description": "Unauthorized: Token has not found or invalid format or session is invalid"
```
[the remark this answers is not in the document]
Comment (namrata-khake-ui) [reply web/src/config/swagger.json:3072]: Added 403 for non public API

[the remark this answers is not in the document]
Comment (namrata-khake-ui) [reply web/src/config/swagger.json:3072]: Added 403 for non public API

File: web/src/config/swagger.json (hunk 1 above)
Comment (namrata-khake-ui) [reply web/src/config/swagger.json:3277]: resolved

**Pull request closed without merging**